packages/SwingSet/src/devices/loopbox.js

```diff
@@ -0,0 +1,8 @@
+export function buildLoopbox() {
```

Comment:
if we omit loopbox, we can also avoid making these two changes to `packages/SwingSet`